plugins/module_utils/redfish_utils.py

```diff
@@ -2792,3 +2800,80 @@ def set_manager_nic(self, nic_addr, nic_config):
         if response['ret'] is False:
             return response
         return {'ret': True, 'changed': True, 'msg': "Modified Manager NIC"}
+
+    def get_hostinterfaces(self):
+        result = {}
+        hostinterface_results = []
+        key = "PowerSupplies"
+        properties = ['Id', 'Name', 'Description', 'HostInterfaceType', 'Status',
```

Comment:
Some other properties to add (they might not be supported by the system you have, but they're useful to report if available):
* AuthNoneRoleId
* CredentialBootstrapping